### DISCUSSION TITLE
GraphQL Mesh with "@envelop/statsd" package

### DISCUSSION THREAD

**Discussion by vijayakumarit** [dotansimha/envelop#1455] [Q&A] [answered]
Hi @n1ru4l 

I need some more input from your end. I have already worked with GraphQL Mesh and Hive for data monitoring as well. I need to implement some logging and datadog operations in Mesh. Can you please help me? How can I use mesh with the "envelop/statsd" package? Do we have any examples of that?

**Comment by n1ru4l** [comment, marked as answer]
@vijayakumarit We currently don't have an example of using statsd together with graphql mesh. You can find the documentation for using envelop plugins together with mesh on https://www.graphql-mesh.com/docs/getting-started/customize-mesh-server#configure-and-provide-envelop-plugins. Feel free to provide a pull request with a usage example to the mesh git repository.

**Reply by vijayakumarit** [reply]
@n1ru4l Thanks for the answer. It's working well. I need some more clarification for Datadog monitoring. 

I have seen the code using the "statsd" package. If I need to implement datadog, what kind of steps do I need to follow? Shall I use the same "statsd" package for that one as well?

What do I need to configure below?
  const client = new StatsD({
    port: 8081,
    globalTags: { env: 'process.env.' },
    });
Shall I parse the mesh config items on the top. like (PORT: 4000) and (globalTags: "http://0.0.0.0:4000/") Is this correct?

For Datadog, do I need to add any additional .YAML files?
